src/docs/ftw-content/hosted-translations/index.md

```diff
@@ -0,0 +1,236 @@
+     to front end.
+   - Note: FTW does not pass the translations themselves from the server
+     to browser. The reason is that if browser fetches the versioned
+     asset file directly, it can leverage browser's cache. So, every
+     page load, after the initial one, will use the `translation.json`
+     file from browsers local cache.
+
+   ```js
+     .then(() => {
+       return { preloadedState: store.getState(), translations };
+     })
+   ```
+
+6. SSR: call `renderApp` function, which renders the `<ServerApp>` in
+   _src/app.js_
+
+   - Hosted translations from Asset Delivery API are passed as props to
+     `ServerApp`.
+
+   ```shell
+   └── server
+       └── renderer.js
+   ```
+
+   ```js
+   // Render the app with given route, preloaded state, hosted translations.
+   const { head, body } = renderApp(
+     requestUrl,
+     context,
+     preloadedState,
+     translations,
+     collectWebChunks
+   );
+   ```
+
+   ```shell
+   └── src
+       └── app.js
+   ```
+
+   ```js
+   <ServerApp
+     url={url}
+     context={serverContext}
+     helmetContext={helmetContext}
+     store={store}
+     hostedTranslations={hostedTranslations}
+   />
+   ```
+
+- Hosted translations are then merged with the default translations in
+  `ServerApp` component.
+  ```jsx
+  export const ServerApp = props => {
+    const { url, context, helmetContext, store, hostedTranslations = {} } = props;
+    setupLocale();
+    HelmetProvider.canUseDOM = false;
+    return (
+      <IntlProvider
+        locale={config.locale}
+        messages={{ ...localeMessages, ...hostedTranslations }}
+        textComponent="span"
+      >
+    {/* etc. */}
+  ```
+
+7. Browser: initialize the store with a preloaded state
+   - The asset version that the SSR used, is included in that preloaded
+     state
+8. Browser: **fetchAppAssets** thunk fetch assets using asset version:
+   calls `sdk.assetByVersion`.
+9. Browser: make _loadData_ call
+10. Browser: hydrate the `<ClientApp>` and pass translations as props.
+
+- Hosted translations are merged with default translations in
+  `ClientApp` component.
+  ```js
+  <ClientApp store={store} hostedTranslations={translations} />
+  ```
+
+## How development build works with hosted translations
+
+This is setup is in use if you run **_yarn run dev_** on local machine.
+
+1. Browser: initialize the store
+2. Browser: **fetchAppAssets** thunk fetches assets using 'latest'
+   alias: calls `sdk.assetByAlias`
+   - Because SSR is not available to fetch the latest version of the
+     asset files, this call needs to be made from browser.
+3. Browser: make _loadData_ call
+4. Browser: render the `<ClientApp>` and pass translations as props.
+
+## Read more
+
+If you want to read more, here are some pointers:
+
+- [Asset Delivery API](/references/<the-correct-link-here>/)
+- [Short intro to SSR](https://deploy-preview-565--sharetribe-flex-docs-site.netlify.app/docs/ftw/how-routing-works-in-ftw/#a-brief-introduction-to-ssr)
```

Comment:
Should this be pointing to a deploy preview?

Reply:
Fixed to an internal link